governance/SMV/README.md

```diff
@@ -0,0 +1,250 @@
+# Working with SMV contracts
+
+## 1. Introduction
+
```

Comment:
We probably need to explain how this whole thing works, get our diagram here, etc. I will add it probably after the review.